.github/workflows/eslint.yml

```
@@ -0,0 +1,15 @@
+name: eslint
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install modules
+      run: npm i
+    - name: Run ESLint on binderhub JS
+      run: npm run lint
```

```suggestion
      run: npm run lint

```

Nit detail, end of line.